beaconChain/tests/helpers/stateTransitionHelpers.test.ts

```diff
@@ -323,3 +324,59 @@ describe("isActiveValidator", () => {
     assert.isFalse(result);
   });
 });
+
+describe("getForkVersion", () => {
+  const fork: Fork = {
+    epoch: new BN(12),
+    previousVersion: new BN(4),
+    currentVersion: new BN(5)
+  };
+
+  const four: uint64 = new BN(4);
+  const five: uint64 = new BN(5);
+
+  it("fork version should be 4", () => {
+    const result = getForkVersion(fork, new BN(8));
+    assert(result.eq(four));
+  });
+
+  it("fork version should be 5", () => {
+    const result = getForkVersion(fork, new BN(13));
+    assert(result.eq(five));
+  });
+
+  it("fork version should be 5", () => {
+    const result = getForkVersion(fork, new BN(12));
+    assert(result.eq(five));
+  });
+});
+
+describe("getDomain", () => {
+  const fork: Fork = {
+    epoch: new BN(12),
+    previousVersion: new BN(4),
+    currentVersion: new BN(5)
+  };
+
+  const constant: uint64 = new BN(2 ** 32);
+  const four: uint64 = new BN(4);
+  const five: uint64 = new BN(5);
+
+  it("domain version should be 4", () => {
+    const result = getDomain(fork, new BN(8),4);
```

Comment:
Probably best to use the same domainType across these tests since we seem to be testing similarish things to getForkVersion

Also spacing of arguments here and below.

Reply:
> Probably best to use the same domainType across these tests since we seem to be testing similarish things to getForkVersion

Not sure what you mean?

Reply:
In the other tests for getDomain, we're using 5 as the domainType, but in this test, we're using 4.
I just thought it would be better to use 5 here as well, but its such a minor issue, doesn't particularly matter.